### PR TITLE
feat: added alert and button components

### PR DESCRIPTION
**Pull request by adidoshi**
Hey! I create the alert & button component.
Live preview for button -
https://deploy-preview-2--splashui.netlify.app/components/button/button

Live preview for alert -
https://deploy-preview-2--splashui.netlify.app/components/alert/alert

### CODE REVIEW

File: components/alert/Alert.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    <link rel="stylesheet" href="/css/sidebar.css" />
+    <link rel="stylesheet" href="/css/main.css" />
+    <link rel="stylesheet" href="/components/alert/Alert.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header-left">
+          <a href="../../index.html">Sparrow</a>
+          <span class="small-text">v1.0</span>
```
Comment (MiheerTamkhane) [inline components/alert/Alert.html:29]: I think you forgot to change your name here?

File: components/button/Button.html [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    <link rel="stylesheet" href="/css/sidebar.css" />
+    <link rel="stylesheet" href="/css/main.css" />
+    <link rel="stylesheet" href="/components/button/Button.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header-left">
+          <a href="../../index.html">Sparrow</a>
+          <span class="small-text">v1.0</span>
```
Comment (MiheerTamkhane) [inline components/button/Button.html:29]: Here too.

File: components/button/Button.html [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+          <div class="example-showcase">
+            <button class="btn btn-link"><a href="#">Link</a></button>
+          </div>
+
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%2843%2C43%2C43%2C1%29&t=seti&wt=none&l=htmlmixed&width=800&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cbutton%2520class%253D%2522btn%2520btn-link%2522%253E%253Ca%2520href%253D%2522link%2522%253ELink%253C%252Fa%253E%253C%252Fbutton%253E"
+            style="
+              width: 800px;
+              height: 76px;
```
Comment (MiheerTamkhane) [inline components/button/Button.html]: I have tried carbon but don't like it. If you can use highlight.js or if you go with carbon add every iframe width: 100% and default hight. its make much more responsive.

File: components/button/Button.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+.btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-primary:hover {
+  background-color: var(--secondary-color);
+}
+
+.btn-secondary {
+  background-color: gray;
```
Comment (mimansha-swarup) [inline components/button/Button.css]: make variable for this too

**Review by pritam-kr — APPROVED** [review]

File: css/header.css [hunk 5]
```diff
@@ -1,34 +1,91 @@
...
+    margin: 0 0.5rem;
+  }
+
+  .header-right li a {
+    font-size: 0.8rem;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .container {
```
Comment (MiheerTamkhane) [inline css/header.css:84]: I think you need to change order too, add media queries from smaller to bigger viewport size. the opposite may cause some problems later. Idk just lil suggession.

**Review by MiheerTamkhane — APPROVED** [review]
Other than that, Its looks good to me.

File: components/alert/Alert.html (hunk 1 above)
Comment (adidoshi) [inline components/alert/Alert.html]: Was looking at reference, changed. Thanks.

File: components/button/Button.html (hunk 2 above)
Comment (adidoshi) [inline components/button/Button.html]: Changed thanks.

**Review by mimansha-swarup — COMMENTED** [review]
all looks good except that

File: components/button/Button.css (hunk 4 above)
Comment (adidoshi) [reply components/button/Button.css]: Thanks, will make the change.

File: css/header.css (hunk 5 above)
Comment (adidoshi) [reply css/header.css:84]: Will look into this, thanks

File: components/button/Button.html (hunk 3 above)
Comment (adidoshi) [reply components/button/Button.html]: Ya carbon causes issues sometimes, thanks for the idea, will make changes.

**Pull request merged**